drivers/axi_core/jesd204/axi_jesd204_rx.c

```diff
@@ -468,7 +478,7 @@ int32_t axi_jesd204_rx_apply_config(struct axi_jesd204_rx *jesd,
 	octets_per_multiframe = config->frames_per_multiframe *
 				config->octets_per_frame;
 
-	multiframe_align = 1 << jesd->data_path_width;
+	multiframe_align = jesd->data_path_width;
```

Comment:
does these changes affect other jesd dependent projects/drivers?

Reply:
Up.

Reply:
Yes, this is the correct  way to assign "multiframe_align" variable